<launch>

  <!-- Launch ROS Package Template Node -->
  <node pkg="husky_highlevel_controller" type="husky_highlevel_controller" name="husky_highlevel_controller" output="screen">
    <rosparam command="load" file="$(find husky_highlevel_controller)/param/param.yaml"/>
  </node>

  <include file="$(find husky_gazebo)/launch/husky_playpen.launch" />

  <node pkg="rviz" type="rviz" name="rviz" output="screen"/>

</launch>